### ISSUE TITLE
Test 230316.001.8at

### ISSUE THREAD

**Issue by sailist** [sailist/image-classification#12]
|Experiment Name|Test Name|Start| End |
|---|---|---|---|
|track_semi.crossentropy|230316.001.8at||23-03-16-095945|

```bash
python train_semi.py --module=crossentropy --config=config/semi/ssl_eval/cifar100.yml --n_percls=100 --scan=ssl-semi-eval-2023.03.04 --pretrain_path=/home/share/yanghaozhe/.lumo/blob/simclr.simclrexp/230314.000.a3t/models/last_model.ckpt --model=wrn282 --device=3 --group=1621
```

## Parameters
```python
{'apply_da': False,
 'batch_count': 0,
 'batch_size': 64,
 'config': 'config/semi/ssl_eval/cifar100.yml',
 'dataset': 'cifar100',
 'device': 3,
 'ema': False,
 'epoch': 400,
 'eval': {'batch_size': 512,
          'num_workers': 8,
          'pin_memory': True,
          'shuffle': True},
 'group': 1621,
 'method': 'baseline',
 'mixup_beta': 0.5,
 'model': 'wrn282',
 'module': 'crossentropy',
 'mu': 1,
 'n_classes': 100,
 'n_percls': 100,
 'optim': {'lr': 0.03,
           'momentum': 0.9,
           'name': 'SGD',
           'nesterov': True,
           'weight_decay': 0.001},
 'pretrain': True,
 'pretrain_path': .lumo/blob/simclr.simclrexp/230314.000.a3t/models/last_model.ckpt',
 'pseudo_thresh': 0.95,
 'record_matrix': True,
 'record_predict': True,
 'scan': 'ssl-semi-eval-2023.03.04',
 'seed': 1,
 'sharpen': 0.5,
 'stl10_unlabeled': True,
 'test': {'batch_size': 512,
          'num_workers': 8,
          'pin_memory': True,
          'shuffle': False},
 'train': {'batch_size': 512,
           'num_workers': 8,
           'pin_memory': True,
           'shuffle': True},
 'with_coarse': False}
```

## Metrics

|Key|Value|
|--|--|
|Acc|0.60100|

Tensor metrics:
```
cm
None

```

```

```

## Full properties
<details>
<summary> Click Here </summary>
<code>
{'acc': 0.601,
 'agent': {'last_edit_time': '23-03-16-063341'},
 'backup': {'23-03-17-165242': {'backend': 'github',
                                'number': 12,
                                'repo': 'pytorch-lumo/image-classification'}},
 'dataset': 'cifar100',
 'deprecated': nan,
 'end': nan,
 'exception': nan,
 'execute': {'cwd': '~/python/image-classification-private',
             'exec_argv': ['train_semi.py',
                           '--module=crossentropy',
                           '--config=config/semi/ssl_eval/cifar100.yml',
                           '--n_percls=100',
                           '--scan=ssl-semi-eval-2023.03.04',
                           .lumo/blob/simclr.simclrexp/230314.000.a3t/models/last_model.ckpt',
                           '--model=wrn282',
                           '--device=3',
                           '--group=1621'],
             'exec_bin': '~/miniconda3/bin/python',
             'exec_file': 'train_semi.py',
             'repo': '~/python/image-classification-private'},
 'exp_name': 'track_semi.crossentropy',
 'git': {'commit': '5a9b62c2',
         'dep_hash': 'a8d8646bd11680716750a88fd641e1ee',
         'repo': '~/python/image-classification-private'},
 'hooks': {'Diary': {'loaded': True, 'msg': ''},
           'FinalReport': {'loaded': True, 'msg': ''},
           'GitCommit': {'loaded': True, 'msg': ''},
           'LastCmd': {'loaded': True, 'msg': ''},
           'LockFile': {'loaded': True, 'msg': ''},
           'RecordAbort': {'loaded': True, 'msg': ''}},
 'lock': {'accelerate': '0.16.0',
          'decorator': '5.1.1',
          'fire': '0.5.0',
          'hydra': '1.3.1',
          'joblib': '1.2.0',
          'lumo': '0.15.0',
          'numpy': '1.24.2',
          'omegaconf': '2.3.0',
          'psutil': '5.9.4',
          'torch': '1.8.1+cu101',
          'torch.version.cuda': '10.1'},
 'logger_args': nan,
 'metric_board': nan,
 'model': 'wrn282',
 'n_percls': 100,
 'note': '',
 'params': {'apply_da': False,
            'batch_count': 0,
            'batch_size': 64,
            'config': 'config/semi/ssl_eval/cifar100.yml',
            'dataset': 'cifar100',
            'device': 3,
            'ema': False,
            'epoch': 400,
            'eval': {'batch_size': 512,
                     'num_workers': 8,
                     'pin_memory': True,
                     'shuffle': True},
            'group': 1621,
            'method': 'baseline',
            'mixup_beta': 0.5,
            'model': 'wrn282',
            'module': 'crossentropy',
            'mu': 1,
            'n_classes': 100,
            'n_percls': 100,
            'optim': {'lr': 0.03,
                      'momentum': 0.9,
                      'name': 'SGD',
                      'nesterov': True,
                      'weight_decay': 0.001},
            'pretrain': True,
            'pretrain_path': .lumo/blob/simclr.simclrexp/230314.000.a3t/models/last_model.ckpt',
            'pseudo_thresh': 0.95,
            'record_matrix': True,
            'record_predict': True,
            'scan': 'ssl-semi-eval-2023.03.04',
            'seed': 1,
            'sharpen': 0.5,
            'stl10_unlabeled': True,
            'test': {'batch_size': 512,
                     'num_workers': 8,
                     'pin_memory': True,
                     'shuffle': False},
            'train': {'batch_size': 512,
                      'num_workers': 8,
                      'pin_memory': True,
                      'shuffle': True},
            'with_coarse': False},
 'params.yaml': nan,
 'params_hash': nan,
 'paths': {'blob_root': .lumo/blob',
           'cache_root': '/home/share/yanghaozhe/.cache/lumo',
           'info_root': .lumo/experiments'},
 'pinfo': {'hash': 'e951610e5fd2e887e7e4fe60c1b19a19',
           'obj': {'argv': ['~/miniconda3/bin/python',
                            'train_semi.py',
                            '--module=crossentropy',
                            '--config=config/semi/ssl_eval/cifar100.yml',
                            '--n_percls=100',
                            '--scan=ssl-semi-eval-2023.03.04',
                            .lumo/blob/simclr.simclrexp/230314.000.a3t/models/last_model.ckpt',
                            '--model=wrn282',
                            '--device=3',
                            '--group=1621'],
                   'pid': 27377,
                   'pname': 'python',
                   'pstart': 1678910416.05},
           'pid': 27377},
 'progress': {'end': '23-03-16-095945',
              'end_code': -10,
              'msg': 'ended by watcher'},
 'rerun': nan,
 'start': nan,
 'state': nan,
 'table_row': nan,
 'tags': [],
 'tensorboard_args': nan,
 'test_name': '230316.001.8at',
 'trainer': {'board_fn': .lumo/blob/track_semi.crossentropy/230316.001.8at/board.sqlite/metric_board.sqlite',
             'logger_fn': .lumo/experiments/track_semi.crossentropy/230316.001.8at/l.0.2303160400.log',
             'params_meta': {'fn': .lumo/experiments/track_semi.crossentropy/230316.001.8at/params.yaml',
                             'hash': 'a977736566aa45978d335acf45a08edf'}}}
</code>
</details>
